.dockerignore

```diff
@@ -1,11 +1,13 @@
 *
+!/ci-scripts
 !/chain-abci
 !/chain-core
-!/chain-tx-enclave
-/chain-tx-enclave/tx-validation/bin
-!/chain-tx-enclave/tx-validation/bin/.gitignore
+!/chain-tx-enclave/tx-validation/bin/enclave.signed.so
```

Comment:
Why include this file?

Reply:
because there is a file called `integration.enclave` which will cause some error when build docker image, so here I just get what I what.

Reply:
`integration.enclave` is a unix socket used in `sgx-cargo-1804-hw2` / tx-query-hw-test.sh

Reply:
I think it's because we are using a pattern like `docker run -v ... build.sh` to build stuff, the intermediate building result is generated in host machine, that's why we have to include them in docker ignore, because when we want to copy them into another docker image?
I think the better practice would be to just store the build result in the result image, and use the image or copy from the image in after operations?

Reply:
AFAIK it depends on if the image is used for building / testing only (e.g. for  tx-query-hw-test.sh it also runs the mini-test after building and the execution is in `docker run --rm ...`, so it's cleaned-up afterwards) or deployment ... for this launcher, I'm a bit puzzled